js/majors.js

```diff
@@ -746,9 +746,12 @@ window.majors = {
             "on the EECS website and 6.AcAd on Piazza.",
         skip: 1
       },
-      [1, [2, [1, "6.01", "6.S08"],
+      [1, [2, "6.0001",
```

Comment:
My style rules for these (since you asked) are:

1) 80-character line limit
2) Matching square/curvy brackets are either on the same line `{foo:"bar"}` or at the same level of indentation if over multiple lines:
```
[2, 
  "foo", "bar",
  {id:"baz",skip:1}
]
```
3) For now, no trailing commas (compatibility issue)

If you see parts of this file where (2) is violated, it's because I did some auto-formatting work in 2013 and haven't brought the whole file into accordance.